src/components/ExpandedBankTransactionRow/ExpandedBankTransactionRow.tsx

```diff
@@ -82,435 +70,347 @@ export interface DocumentWithStatus {
+                                showLabels={false}
+                                isReadOnly={!categorizationEnabled}
+                                className={`${className}__table-cell--split-entry__tags`}
                               />
-                              {showTags && (
-                                <TagDimensionsGroup
-                                  value={split.tags}
-                                  onChange={tags => changeTags(index, tags)}
-                                  showLabels={false}
+                            )}
+                            {showCustomerVendor && (
+                              <div className='Layer__expanded-bank-transaction-row__table-cell--split-entry__customer'>
+                                <CustomerVendorSelector
+                                  selectedCustomerVendor={split.customerVendor}
+                                  onSelectedCustomerVendorChange={customerVendor => changeCustomerVendor(index, customerVendor)}
+                                  placeholder='Set customer or vendor'
                                   isReadOnly={!categorizationEnabled}
-                                  className={`${className}__table-cell--split-entry__tags`}
+                                  showLabel={false}
                                 />
-                              )}
-                              {showCustomerVendor && (
-                                <div className='Layer__expanded-bank-transaction-row__table-cell--split-entry__customer'>
-                                  <CustomerVendorSelector
-                                    selectedCustomerVendor={split.customerVendor}
-                                    onSelectedCustomerVendorChange={customerVendor => changeCustomerVendor(index, customerVendor)}
-                                    placeholder='Set customer or vendor'
-                                    isReadOnly={!categorizationEnabled}
-                                    showLabel={false}
-                                  />
-                                </div>
-                              )}
-                              <div className='Layer__expanded-bank-transaction-row__table-cell--split-entry__button'>
-                                <Button
-                                  onPress={() => removeSplit(index)}
-                                  variant='outlined'
-                                  icon
-                                  isDisabled={index == 0}
-                                >
-                                  <Trash size={18} />
-                                </Button>
                               </div>
+                            )}
+                            <div className='Layer__expanded-bank-transaction-row__table-cell--split-entry__button'>
+                              <Button
+                                onPress={() => removeSplit(index)}
```

Comment:
Personally think this is fine, trivial to re-render Button.

_I'll eventually not comment about this, but want to point and share my thoughts in case you have a differing opinion!_